ControlRoomApplication/ControlRoomApplication/GUI/RTControlForm.cs

```diff
@@ -1081,5 +1081,10 @@ private void speedTextBox_TextChanged(object sender, EventArgs e)
         {
 
         }
+
+        private void FreeControlForm_Load(object sender, EventArgs e)
+        {
+
+        }
```

Comment:
I mentioned this in another [PR](https://github.com/YCPRadioTelescope/YCP-RT-ControlRoom/pull/440). These empty event handlers should probably be removed.

- [ ] Remove empty event handler